holoviews/element/chart.py

```diff
@@ -5,7 +5,7 @@
 from ..core import util
 from ..core import Dimension, Dataset, Element2D
 from ..core.data import GridInterface
-from .geom import Points, VectorField # noqa: backward compatible import
+from .geom import Points, Geometry, Segments, VectorField # noqa: backward compatible import
```

Comment:
The others are just listed for backward compatibility.

```suggestion
from .geom import Points, VectorField # noqa: backward compatible import
```

Reply:
In that case we'd need a `from .geom import *` in `__init__.py`, unless I'm missing something.

Reply:
It should be imported into ``element/__init__.py``, there's no need to expose it in ``element/chart.py``.

Reply:
Yes - but any sort of import from `.geom` has been missing from `element/__init__.py` so as far as I can see the import of VectorField has been relying on exposing it in `chart.py` first.

Reply:
Sure, we should just add that.

Reply:
Done